sdk/python/foundation-models/system/finetune/Llama-notebooks/text-generation/text-generation-samsum.ipynb

```diff
@@ -0,0 +1,794 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Text Generation - SamSum \n",
+    "\n",
+    "This sample shows how use `text-generation` components from the `azureml` system registry to fine tune a model to summarize a dialog between 2 people using samsum dataset. We then deploy the fine tuned model to an online endpoint for real time inference.\n",
+    "\n",
+    "### Training data\n",
+    "We will use the [samsum](https://huggingface.co/datasets/samsum) dataset. This dataset is intended to summarize dialogues between 2 people. with this notebook we will summarize the dialogues and calculate bleu and rouge scores for the summarized text vs provided ground_truth summaries\n",
+    "\n",
+    "### Model\n",
+    "We will use the `llama-2-7b` model to show how user can finetune a model for text-generation task. If you opened this notebook from a specific model card, remember to replace the specific model name. Optionally, if you need to fine tune a model that is available on HuggingFace, but not available in `azureml` system registry, you can either [import](https://github.com/Azure/azureml-examples) the model or use the `huggingface_id` parameter instruct the components to pull the model directly from HuggingFace. \n",
```

Comment:
Any idea why the import is pointed to azureml-examples?

Reply:
I see the same for another task as well. I think this is the right link - https://github.com/Azure/azureml-examples/blob/main/sdk/python/foundation-models/system/import/import_model_into_registry.ipynb

Reply:
yeah, changed for text gen now, will pick up other tasks later